pkg/kubelet/allocation/features_linux.go

```diff
@@ -35,3 +35,10 @@ func IsInPlacePodVerticalScalingAllowed(pod *v1.Pod) (allowed bool, msg, reason
 	}
 	return true, "", ""
 }
+
+func IsInPlacePodLevelResourcesVerticalScalingAllowed(pod *v1.Pod) (allowed bool, msg, reason string) {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
+		return false, "InPlacePodLevelResourcesVerticalScaling is disabled", "feature_gate_off"
```

Comment:
nit: the reason is missing the context
```suggestion
		return false, "InPlacePodLevelResourcesVerticalScaling is disabled", "plr_feature_gate_off"
```